src/device-registry/models/Device.js

```diff
@@ -483,6 +483,12 @@ deviceSchema.statics = {
           foreignField: "_id",
           as: "cohorts",
         })
```

Comment:
Just added the grids to the details returned when one queries device details. @BenjaminSsempala